tests/unit/dataactvalidator/test_b20_object_class_program_activity.py

```diff
@@ -15,42 +15,69 @@ def test_column_headers(database):
 
```

Comment:
I can't actually comment higher than this, but the `&` doesn't have to be there. You can just do `actual == expected_subset` with the same results (actually more accurate). Because this doesn't include all the columns we should be receiving back as-is